### ISSUE TITLE
Invalid Status attributes appendlimit

### ISSUE THREAD

**Issue by luckman212** [gonejack/import-mail#2]
Hi again @gonejack -- sorry to bother but issue #1 has returned for me with latest version 1.0.4
IMAP server is FastMail.

I had to revert back to v1.0.1 to get it to work again

```
2022/04/04 07:52:59 Invalid Status attributes appendlimit
```

**Comment by gonejack** [comment]
Hi, @luckman212 

This is wild since there are no printing actions on this error in v1.0.4

Could you try download prebuiuld binaries from [Release](https://github.com/gonejack/import-mail/releases/tag/v1.0.4)

**Comment by luckman212** [comment]
Thanks @gonejack Hmm yes something is weird...

When I download the release manually, it works and I see the syntax has changed, new help text etc.

But, when I install with `go install github.com/gonejack/import-mail@latest` for example, I end up with what seems to be an old version of the tool. I don't know how this is possible? But I guess it's related to the [go.mod](https://github.com/gonejack/import-mail/blob/main/go.mod) file somehow? Maybe there is a bad reference there.

**Comment by gonejack** [comment]
It could happens sometime by some caching issues in my experiences. 

In this case a `git clone` and `go build .` would be helpful.

**Comment by luckman212** [comment]
Yes, cloning the repo and building works. But the module versions are all different vs what you get from go install. So something is definitely off (I even tried clearing my build cache)

### Installed via `go install`
```shell
$ go clean -cache -modcache
$ go install github.com/gonejack/import-mail@latest
go: downloading github.com/gonejack/import-mail v1.0.4
go: downloading github.com/spf13/cobra v1.1.3
go: downloading github.com/emersion/go-imap v1.0.6
go: downloading github.com/emersion/go-imap-appendlimit v0.0.0-20190308131241-25671c986a6a
go: downloading github.com/dustin/go-humanize v1.0.0
go: downloading github.com/spf13/pflag v1.0.5
go: downloading github.com/emersion/go-sasl v0.0.0-20191210011802-430746ea8b9b
go: downloading golang.org/x/text v0.3.2
```

### Cloned and manually built from repo:
```shell
$ go build .
go: downloading github.com/alecthomas/kong v0.2.22
go: downloading github.com/emersion/go-imap v1.2.0
go: downloading github.com/emersion/go-imap-appendlimit v0.0.0-20210907172056-e3baed77bbe4
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/emersion/go-sasl v0.0.0-20200509203442-7bfe0ed36a21
go: downloading golang.org/x/text v0.3.7
```

**Comment by gonejack** [comment]
I can't remember is there any mistake tagging of `v1.0.4` months ago, anyway new release `v1.0.5` should solve this.

**Comment by luckman212** [comment]
Thanks, yes confirming that 1.0.5 works perfectly !
